tests/cephfs/snapshot_clone/cg_snap_system_test.py

```diff
@@ -0,0 +1,765 @@
+            "vol_name": default_fs,
+            "group_name": "subvolgroup_cg",
+        }
+        fs_util_v1.create_subvolumegroup(client1, **subvolumegroup)
+        qs_cnt += 1
+        sv_def_list = []
+        sv_non_def_list = []
+        for i in range(1, qs_cnt):
+            sv_name = f"sv_def_{i}"
+            sv_def_list.append(sv_name)
+            subvolume = {
+                "vol_name": default_fs,
+                "subvol_name": sv_name,
+                "size": "64424509440",
+            }
+            fs_util_v1.create_subvolume(client1, **subvolume)
+        for i in range(1, qs_cnt):
+            sv_name = f"sv_non_def_{i}"
+            subvolume = {
+                "vol_name": default_fs,
+                "subvol_name": sv_name,
+                "group_name": "subvolgroup_cg",
+                "size": "64424509440",
+            }
+            fs_util_v1.create_subvolume(client1, **subvolume)
+            sv_name = f"subvolgroup_cg/sv_non_def_{i}"
+            sv_non_def_list.append(sv_name)
+
+        client1.exec_command(
+            sudo=True,
+            cmd="ceph config set mds log_to_file true;ceph config set mds debug_mds_quiesce 10",
+            check_ec=False,
+        )
+        sv_mixed_list = []
+        qs_cnt -= 1
+        for i in range(0, qs_cnt):
+            sv_name1 = random.choice(sv_def_list)
+            sv_name2 = random.choice(sv_non_def_list)
+            if (sv_name2 not in sv_mixed_list) and (len(sv_mixed_list) < qs_cnt):
+                sv_mixed_list.append(sv_name2)
+            if (sv_name1 not in sv_mixed_list) and (len(sv_mixed_list) < qs_cnt):
+                sv_mixed_list.append(sv_name1)
+            if len(sv_mixed_list) == qs_cnt:
+                break
+
+        qs_sets = [
+            sv_def_list,
+            sv_non_def_list,
+            sv_mixed_list,
+        ]
+        log.info(f"Test config attributes : qs_cnt - {qs_cnt}, qs_sets - {qs_sets}")
+        cg_test_params = {
+            "ceph_cluster": ceph_cluster,
+            "fs_name": default_fs,
+            "fs_util": fs_util_v1,
+            "cg_snap_util": cg_snap_util,
+            "cg_snap_io": cg_snap_io,
+            "clients": clients,
+            "mgr_node": mgr_node,
+            "qs_sets": qs_sets,
+            "cg_run_time": 3600,
+        }
+        for test_name in test_list:
+            log.info(
+                f"\n\n                                   ============ {test_name} ============ \n"
+            )
+            cg_test_params.update({"test_case": test_name})
+            test_status = cg_system_test(cg_test_params)
+
+            if test_status == 1:
+                assert False, f"Test {test_name} failed"
+            else:
+                log.info(f"Test {test_name} passed \n")
+        return 0
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Clean Up in progess")
```

Comment:
Can you please check finally block? I do not see any nfs cluster nad export created in the test.

Reply:
done